.claude/commands/work-on-issue.md

```
@@ -9,14 +9,56 @@ Execute the gated workflow for issue #$ARGUMENTS.
+
+**Before anything else**, check for existing workflow state:
+
+1. Check for existing workflow:
+
+   ```bash
+   bun scripts/checkpoint-cli.ts find $ARGUMENTS
+   ```
+
+2. **If existing workflow found** (not null):
+   - Show the workflow state:
+     ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)